practice-two/src/index.html

```diff
@@ -51,7 +51,37 @@ <h1 class="navbar__logo">H.</h1>
         </nav>
 
         <!-- START BANNER -->
-
+        <section class="banner-section">
+          <hgroup class="banner-section__header">
+            <div class="banner-section__header-intro">
+              <p class="banner-section__header-text">RESPONSIVE UI KIT</p>
+            </div>
+            <h1 class="banner-section__header-title">HOFMANN</h1>
```

Comment:
Should group banner-section__header-title into banner-section__description instead of group it with banner-section__header-text